roseus/euslisp/roseus-utils.l

```diff
@@ -616,6 +616,20 @@
              cb)) points)))
     ))
 ;;
+;; for boudingbox
```

Comment:
typo

Reply:
Thank you for your review.
I fixed the spell.